requirements.txt

```diff
@@ -1,6 +1,7 @@
 click>=7.1
 colorama>=0.3
 configparser
+cached-property

```

Comment:
I'm guessing this is responsible for a good portion of the speed boost?

Reply:
Yes. That and the other caching within the grammars which I've had to implement slightly differently.